### DISCUSSION TITLE
Windows Server 2022?

### DISCUSSION THREAD

**Discussion by plancked** [apohl79/audiogridder#926] [Help]
Hi!

Does Audiogridder server run on Windows Server 2022?
I have installed it and it works, but I do not get a GUI on the client machine.

Thanks
Alex

**Comment by jaacMusic** [comment]
Hi,
Is probably because you don't have a monitor connected to the Server system. One way around this if you plan not to have a monitor/video card on the server is to install a virtual video card driver.
Hope that helps!
